__tests__/base.js

```diff
@@ -1214,7 +1234,7 @@ function runBaseTest(name, autoFreeze, useStrictShallowCopy, useListener) {
 						expect(result.items[0].value).toBe(999)
 					})
 
-					test("mutation during filter callback", () => {
+					test.skip("mutation during filter callback", () => {
```

Comment:
NIT: let's leave a comment on why this is skipped now and refer to the other test